compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -181,6 +181,10 @@ struct InsertQuantizeOp final : public luci::CircleNodeMutableVisitor<void>
     if (input->opcode() == luci::CircleOpcode::CIRCLECONST)
       return nullptr;
 
+    // input is not quantizable (ex: index)
+    if (input->quantparam() == nullptr)
```

Comment:
This change is necessary to pass mixed-precision quantization test (`TEST_WITH_LAYER_INFO` in `QuantizedModelVerifier.test.cpp`).